src/pages/home/index.js

```diff
@@ -1,14 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { FormattedMessage } from 'react-intl';
 import Link from 'shared/components/link';
 
-const Home = () => (
-    <div>
-        <h1>Hi people</h1>
-        <p>Welcome to your new Gatsby site.</p>
-        <p>Now go build something great <FormattedMessage id="foo" />.</p>
-        <Link to="/page-2/">Go to page 2</Link>
-    </div>
-);
+import { register, unregister } from 'shared/service-worker/register'
+
+class Home extends Component {
+    componentDidMount() {
+        // Force a clean state of the service worker, as user must interact for enabling it
+        unregister();
```

Comment:
This is just for demo purposes right? Shouldn't be this be done at the `layout/index.js` instead of the homepage?

Reply:
Yeah, it is an initial integration of the feature! 

My first implementation was at `layout/index.js`, as you suggest. However, since this part of the PR is not final (design needs to be defined), I decided to centralize all this by now. For example, if the toggle button appears in the header, maybe the unregister should be there as well.

Reply:
what do you think @satazor ?

Reply:
I would say it's okay to keep it here but @PedroMiguelSS must be aware that we should move this to the mount of the layout component.

Reply:
We have been talking about this topic, and we will have the toggle button for register and unregister the Service Worker. This way, the unregister will not need to be in the layout component.